include/picojson/picojson.h

```diff
@@ -117,6 +119,14 @@ extern "C" {
 
 namespace picojson {
```

Comment:
This is a thirdparty library and I am not sure changing the source code here is going to help us move away from it since it's no longer maintained

Reply:
If you need to modify the json library for it to work with jwt-cpp then we can't do so much string_view. jwt-cpp aims to be as agnostic as possible to json/ssl libraries.